### ISSUE TITLE
Same Image Preview

### ISSUE THREAD

**Issue by EazyFTW** [divamgupta/diffusionbee-stable-diffusion-ui#60]
I don't know if this issue was already created, I already looked, but, the image preview, at least for me, continues to show the same image upon sending another request. Although, when saving the image to a file it saves correctly. Any prompt I type in after the first one shows the first prompts image in the preview. I don't believe this has happened before, so I have zero idea why this would occur.

**Comment by VossBC** [comment]
Same. This started just today, and occurs even if I change parameters.

**Comment by wemmick** [comment]
Same here.

**Comment by divamgupta** [comment]
That should not happen. Could you make a video, and ill take a look

**Comment by wemmick** [comment]
@divamgupta Sure: https://drive.google.com/file/d/1P_23a1FB24jYQRxVtPwaOE2ZewHxqLF9/view?usp=sharing

**Comment by divamgupta** [comment]
Thanks a lot! It is caching the image. Thanks for bringing this up. I will fix it.

**Comment by divamgupta** [comment]
The generated images are saved in /tmp/samples.
@wemmick  After generating the 2nd image, do you see both of them?

**Comment by wemmick** [comment]
@divamgupta Nope, just the original image every time. Have to shut down the app and restart to get a new image.

**Comment by jemueller** [comment]
i was getting the same issue earlier today, so i quit and restarted the app and the issue persisted.

i then updated my mac to latest macOS and restarted it and the issue went away. this was before i saw this issue so all i have is this anecdotal evidence

if i see a similar behavior in the future i'll try to examine the /tmp/samples folder to see if i can track it down.

thanks

**Comment by gammons** [comment]
is it because the seed value is always the same?  (e.g. #51)

**Comment by ileff** [comment]
restarting DiffusionBee solved this issue for 1 generation but then it would be stuck again. next i tried rebooting my laptop and that solved the problem. i now see each newly generated image.

**Comment by EazyFTW** [comment]
> I have the same issue. Preview image is the same... BUT... the saved image, is actually... New. prevails, still going on.. After restart, appears to work for a while, strangely, worked fine for several days, but now gets quickly into this preview bug..

Correct, like I mentioned in the issue (I probably could've explained it more clearly), if you save the image, it works fine.

**Comment by VossBC** [comment]
It's overwriting the same file in /tmp/samples every time. For me, it's 00090.png

**Comment by divamgupta** [comment]
Should be fixed in 0.2.0. Please confirm if 0.2.0 works.

**Comment by jemueller** [comment]
havent seen any issues with this since updating to 0.2.0... will let you know if i see the issue recur.

also, where are the temporary images being saved to now? they do not seem to be in /tmp/samples anymore but i was using that location to copy the photos in bulk without having to click the save button for each one in the interface

thank you

**Comment by VossBC** [comment]
It looks like the preview images are now loose in /tmp
